### ISSUE TITLE
Feature pause for 0.9.39

### ISSUE THREAD

**Issue by cmsj** [Hammerspoon/hammerspoon#528]
Since we've changed an awful lot of stuff, particularly core things, since 0.9.38, I think we should try and limit our feature progress until 0.9.39 is out.

I've set up a milestone for 0.9.39 and I'd like everyone to nominate the bugs/PRs they would like to see dealt with before the release, so that we can focus on getting the release tested and out in the wild.

(By "everyone" I mainly mean @asmagill and @lowne , but anyone is welcome to nominate :)

I'd like to get 0.9.39 out in the next week or so, and right now I'm not aware of any issues that would block the release.

**Comment by cmsj** [comment]
To be slightly more clear, I think we should immediately halt landing any additional features, and only nominate issues that we're aware of with the current state of master. I am not currently tracking any regressions at all, so if others could report in swiftly, I could cut a release in the next couple of days, and we can get back to adding cool stuff :)

**Comment by lowne** [comment]
TL;DR: as far as I'm concerned, go ahead with the planned release.

---

No blocking bugs here (off master), the changes to core seem to be working ok.

I haven't *fully* tested some of the high-level additions such as window/application.find (i.e., I haven't tried to make it explode on purpose, but honestly I don't worry too much about that).
Window.layout is still WIP, and there is a (small) chance window.filter will need further breaking API changes to accommodate it; in an ideal world I'd want to avoid these in two consecutive releases, then again the giant disclaimer is there for a reason, so that risk is also not a concern imo.

**Comment by asmagill** [comment]
Off the top of my head I can't think of anything specific that needs to go in this release... freeze away :-)

**Comment by cmsj** [comment]
Awesome. Thanks both!
